apps/coscrad-frontend/src/components/resources/factories/resource-detail-conainer.ts

```diff
@@ -0,0 +1,7 @@
+import { FunctionalComponent } from '../../../utils/types/functional-component';
+
+type MayHaveId = {
```

Comment:
This feels awkward. I'm pretty sure it's not needed any more. Try to at least make the id required where it is used.

Reply:
removed